src/components/search/omnibox-directions.component.tsx

```diff
@@ -38,16 +44,16 @@ import UtilityService from "../../services/utility.service";
  */
 export interface OmniboxDirectionsProps {
   currentLocation: Location;
-  startLocation: MarkerLocation;
-  endLocation: POI;
-  setStartLocation: (marker: MarkerLocation) => void;
-  setEndLocation: (poi: POI) => void;
+  startLocation: MarkerLocation | Building;
+  endLocation: POI | Building;
+  setStartLocation: (marker: MarkerLocation | Building) => void;
+  setEndLocation: (locations: POI | Building) => void;
```

Comment:
In both of these parentheses just write: `(location: MarkerLocation | Building)` to be consistent.

Reply:
Changed `setStartLocation` and `setEndLocation` so they are consistent in commit 8804195